Sources/LiveViewNative/NavStackEntryView.swift

```diff
@@ -60,7 +60,7 @@ struct NavStackEntryView<R: RootRegistry>: View {
                             self.liveViewModel.cachedNavigationTitle = navigationTitle
                         }
                 } else {
-                    fatalError("State is `.connected`, but no `Document` was found.")
+                    R.loadingView(for: coordinator.url, state: .connectionFailed(LiveConnectionError.missingDocument))
```

Comment:
This is a little weird, since this state isn't actually the coordinator state. If the connection setup code were to check that there's a document, and transition to the connectionFailed/missingDocument state if not, I think force unwrapping in the view body is fine?

Reply:
I couldn't actually see a path where this state is possible.